.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "prettier.printWidth": 100
+}
```

Comment:
I don't think we need such file, you can just make prettier default in your vscode and turn on "on save" 
![image](https://user-images.githubusercontent.com/41207896/130080294-4b9e9d5f-2863-4d97-b529-933f2f47e80a.png)